docs/notebooks/multilingual_text_to_image.md

```diff
@@ -12,62 +12,72 @@ jupyter:
     name: python3
 ---
 
-# Multilingual Text-to-Image Search with MultilingualCLIP
+<!-- #region id="72867ba9-6a8c-4b14-acbf-487ea0a61836" -->
+# Multilingual Text-to-Image search with MultilingualCLIP
 
 <a href="https://colab.research.google.com/drive/1N7iWZV0OunFZSLtsQxoazS808MPXhCwq?usp=sharing"><img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg"></a>
 
+<!-- #endregion -->
 
-
+<!-- #region id="f576573b-a48f-4790-817d-e99f8bd28fd0" -->
 Most text-image models are only able to provide embeddings for text in a single language, typically English. Multilingual CLIP models, however, are models that have been trained on multiple different languages. This allows the model to produce similar embeddings for the same sentence in multiple different languages.  
 
 This guide will show you how to finetune a multilingual CLIP model for a text to image retrieval task in non-English languages.
 
 *Note, Check the runtime menu to be sure you are using a GPU/TPU instance, or this code will run very slowly.*
 
+<!-- #endregion -->
 
-
+<!-- #region id="ed1e7d55-a458-4dfd-8f4c-eeb02521c221" -->
 ## Install
+<!-- #endregion -->
 
-```python
-!pip install 'finetuner[full]'
+```python id="9261d0a7-ad6d-461f-bdf7-54e9804cc45d"
+!pip install finetuner
```

Comment:
A code block towards the end uses the `encode` method, do we not need 'finetuner[full]' for that?

Reply:
Good catch! Not sure why this was changes